app/routes/interestgroups/components/InterestGroup.js

```diff
@@ -15,15 +15,14 @@ type Props = {
 };
 
 const InterestGroupComponent = ({ group }: Props) => {
-  const { memberships = [] } = group;
   return (
     <Flex className={styles.listItem}>
       <Flex column className={styles.listItemContent} style={{ flex: '1' }}>
         <Link to={`/interestgroups/${group.id}`} className={styles.link}>
           <h2>{group.name}</h2>
         </Link>
         <div>{group.description}</div>
-        <div>{memberships.length} medlemmer</div>
+        <div>{group.numberOfUsers} medlemmer</div>
```

Comment:
This results in `1` request on `abakus.no/interestgroups`instead of `30` today. 10/10 chance of ssr timeout...

Reply:
:heart_eyes: